### DISCUSSION TITLE
Deadlock when load default scene

### DISCUSSION THREAD

**Discussion by Slond** [NVlabs/sionna#275] [Q&A] [answered]
I'm trying to repeat all examples in documentation of RT, and have some issue:

```
RuntimeError: ​[xml.cpp:215] Error while loading "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\sionna\rt\scenes\munich\munich.xml" (at line 11, col 3): resource deadlock would occur: resource deadlock would occur.
```

![Снимок экрана 2023-12-01 152108](https://github.com/NVlabs/sionna/assets/37597315/218f6cbe-a2fe-44ec-9441-0c1ce87f4718)

**Comment by Slond** [comment, marked as answer]
Found answer:
If someone has this issue, you need to download LLVM 16 version, not 17 or any above

**Reply by alialsahlany** [reply]
Can you explain how to download and activate version 16?

**Reply by merlinND** [reply]
Hello @alialsahlany,

This will depend on your operating system, package manager, etc. This is the official releases page for the LLVM project: https://releases.llvm.org/

**Reply by CHX-7** [reply]
Can I ask which version of Mitsuba you are using?

**Reply by SebastianCa** [reply]
Hi @CHX-7,
I am currently using `3.5.2`. It is required to have `mitsuba >= 3.2.0`.

**Reply by CHX-7** [reply]
> 你好@CHX-7， 我目前正在使用`3.5.2`。需要有`mitsuba >= 3.2.0`。

Thank you.
